docs/troubleshooting.rst

```diff
@@ -276,3 +276,16 @@ the log level of the command to the DEBUG level and the ``-X`` option shows stac
 .. code-block:: sh
 
    $ inmanta -vvv export -X
+
+Debugging
+=========
+
+Debugging the server is possible in case the `rpdb <https://pypi.org/project/rpdb/>`_ package installed.
```

Comment:
We should also explain how this package can be installed. As far as I can see we never explain the documentation where the venv of the server is stored.